packages/stryker/src/utils/TempFolder.ts

```diff
@@ -15,6 +15,7 @@ export class TempFolder {
   initialize(tempDirName = '.stryker-tmp') {
```

Comment:
This variable `tempDirName` should not have a default value, as this is a duplicate from the `Config` class.